algorithms/ar-cochprb/java/coin_change_problem.java

```diff
@@ -0,0 +1,51 @@
+class coinchange{ 
```

Comment:
Codacy found an issue: [The class name 'coinchange' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6342238)